languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/spelling.txt

```diff
@@ -24,17 +24,15 @@
 # Use `\_` to add words with an underscore.
 # Words ending in `_in` will be expanded to have the common gender gap characters: $1*in, $1:in, $1_in, $1*innen, $1:innen, $1_innen
 # The lines in `prefix_verb` syntax will also be used by GermanTagger.
-# For street names please always add both options: e. g. Adalbertstraße and Adalbertstr.
```

Comment:
Könntest du den Kommentar `# /T will add str. (Goethestraße --> Goethestr.)` hier noch einfügen? Dann sieht man es gleich auch in den Instructions :-)

Reply:
@Luisa-LT Ja! Habe ich schon gemacht, das ist nicht mehr im gezeigten Ausschnitt drin, da das in Zeile 22 steht!

Reply:
Dann ist alles super!